nautobot/extras/models/change_logging.py

```diff
@@ -133,6 +133,22 @@ class Meta:
                 name="extras_objectchange_rtime_idx",
                 fields=["-time"],
             ),
+            models.Index(
+                name="changed_object_double_idx",
```

Comment:
The 30 character limit prevents me from using the field naming standard used above, i.e., `extras_objectchange_....`

Reply:
Personally I don't think that `double` or `triple` are particularly useful terms to include in these names, though I'm not a big fan of including `extras_objectchange` in the name either. I might suggest for the new indices:

- `changed_object_idx`
- `related_object_idx`
- `user_changed_object_idx`
- `user_name_changed_object_idx`